components/fonts/platform/freetype/font_list.rs

```diff
@@ -293,7 +293,6 @@ fn font_weight_from_fontconfig_pattern(pattern: *mut FcPattern) -> Option<FontWe
 /// Creates a String from the given null-terminated buffer.
 /// Panics if the buffer does not contain UTF-8.
 unsafe fn c_str_to_string(s: *const c_char) -> String {
```

Comment:
Same question here:

Reply:
Passing a nullptr, or some other invalid pointer to `CStr::from_ptr()` is UB, so we can't make this function safe.

Reply:
Right, since the actual behaviour of the function did not change (no additional checks etc) it remains `unsafe`.